capstone-frontend/src/App.jsx

```diff
@@ -1,14 +1,13 @@
 import React, { Component } from 'react';
 import {
-  Route, 
+  Route,
   BrowserRouter as Router
 } from 'react-router-dom';
 
 import './App.css';
 import Home from './components/Home';
 import Browse from './components/Browse';
 import MyBooks from './components/MyBooks';
-import CreateList from './components/CreateList';
```

Comment:
Is the only way to create a list now through a BookSearchTile? We should make it more obvious that this is where users create lists. Not necessary for this PR, but maybe a separate button on each BookSearchTile to create a new list with that book in it.

Reply:
No, just import it anywhere and provide it ```App.jsx```'s booklists props